contracts/mvx-game-sc/src/owner.rs

```diff
@@ -76,4 +78,16 @@ pub trait OwnerModule: crate::private::PrivateModule + crate::storage::StorageMo
     fn set_game_start_fee(&self, amount: BigUint) {
         self.game_start_fee().set(amount)
     }
+
+    #[only_owner]
+    #[endpoint(setAdmin)]
+    fn set_admin(&self, user: ManagedAddress) {
+        self.is_address_admin(&user).set(true)
```

Comment:
Use Whitelist mapper instead. More efficient, and you can just check if address is whitelisted through the built in function of the mapper, instead of the new `require_address_is_admin` function.